broker-node/app/Clients/requests/IriData.php

```diff
@@ -4,7 +4,7 @@
 
 class IriData
 {
-    public static $nodeUrl = "http://localhost:14265";
+    public static $nodeUrl = "http://165.227.79.113:14265";
```

Comment:
This probably should be localhost.  Usually, the iri requests are going to be coming from within the broker.